EIPS/eip-875.md

```diff
@@ -3,7 +3,8 @@ eip: 875
 title: Simplier NFT standard with batching and native atomic swaps
 author: Weiwu Zhang <a@colourful.land>, James Sangalli <j.l.sangalli@gmail.com>
 discussions-to: https://github.com/ethereum/EIPs/issues/875
-status: Final
+status: Last Call
+review-period-end: 2019-07-01
```

Comment:
I think this is too short of a deadline.

Reply:
If you compare this ERC to other final ERCs (such as 777) it feels a tiny bit under specified for being Final.